libhoney/client.py

```diff
@@ -173,6 +173,19 @@ def close(self):
         # we should error on post-close sends
         self.xmit = None
 
+
+    def flush(self):
+        '''Closes and restarts the transmission, sending all events. Use this
+        if you want to perform a blocking send of all events in your
+        application.
+
+        Note: does not work with asyncronous Transmission implementations such
+        as TornadoTransmission.
+        '''
+        if self.xmit and isinstance(self.xmit, Transmission):
```

Comment:
I'm not in love with this `isinstance` check (I'm not sure of the Pythonic way, but if this was Ruby I'd probably prefer `if self.xmit.respond_to?(:flush)`), but it seems fine while we only have two Transmission implementations (plus the dynamically mocked ones in the tests).

Reply:
We could just have a no-op implementation in the other Transmission - might have less overhead than inspecting the object on every call.